Userland/DevTools/HackStudio/Dialogs/GML/GMLPreviewDialog.cpp

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright (c) 2021, Conor Byrne <cbyrneee@protonmail.com>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#include "GMLPreviewDialog.h"
+#include <LibGUI/BoxLayout.h>
+
+namespace HackStudio {
+
+GMLPreviewDialog::GMLPreviewDialog(String const& content, String const& filename)
```

Comment:
That's essentially a duplicated API, why can't the owner of this widget call `load_gml()` itself? Seems a bit more expressive.

Reply:
That would make more sense, yeah :)